test/Graphing/Calls/Spec.hs

```diff
@@ -6,19 +6,19 @@ import Prelude hiding (readFile)
 import SpecHelpers hiding (readFile)
 
 import Algebra.Graph
-import Data.List (uncons)
 
 import           "semantic" Data.Graph (Graph (..), topologicalSort)
 import           Data.Graph.ControlFlowVertex
 import qualified Data.Language as Language
 import           Semantic.Graph
+import qualified System.Path as Path
 
-callGraphPythonProject :: [FilePath] -> IO (Semantic.Graph.Graph ControlFlowVertex)
-callGraphPythonProject paths = runTaskOrDie $ do
+callGraphPythonProject :: Path.RelFile -> IO (Semantic.Graph.Graph ControlFlowVertex)
+callGraphPythonProject path = runTaskOrDie $ do
```

Comment:
So this is semantically different, but we only ever use it with single paths anyway; is that the idea?

Reply:
Yes.